eparticlectr/eparticlectr.cpp

```diff
@@ -511,8 +722,37 @@ void eparticlectr::oldvotepurge( uint64_t proposal_id, uint32_t loop_limit ) {
     }
 }
 
+[[eosio::action]]
+void eparticlectr::oldvteprgeex( uint64_t proposal_id, uint32_t loop_limit ) {
+    // Get the proposal object
+    propstblex proptable( _self, _self.value );
+    auto prop_it = proptable.find(proposal_id);
+    eosio::check( prop_it == proptable.end() || prop_it->finalized, "Proposal not finalized yet!");
```

Comment:
not sure if this will work as intended. 

calling this action, any user will be able to delete the last proposal even if it didn't finish

Reply:
I thought `prop_it->finalized` would prevent that